Sources/XMTPiOS/Client.swift

```diff
@@ -696,4 +703,22 @@ public final class Client {
 		}
 		return InboxState(ffiInboxState: try await client.inboxState(refreshFromNetwork: refreshFromNetwork))
 	}
+	
+	// See for more details https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-10.md
+	public func isValidAccountID(_ accountAddress: String) -> Bool {
```

Comment:
@neekolas Is this the expected pattern for the address of a SCW?

Reply:
Yup, SCWs are all identified in CAIP-10 format